CMakeLists.txt

```diff
@@ -513,6 +522,9 @@ else()
     set(DILIGENT_INSTALL_PDB OFF)
 endif()
 
+if(PLATFORM_FREEBSD AND EXISTS /usr/local/include)
+	target_include_directories(Diligent-BuildSettings SYSTEM AFTER INTERFACE /usr/local/include)
```

Comment:
What errors do you get without this include?
This change does not seem right

Reply:
It is quite correct.  Without this, anything that isn't part of base will not be found.

Reply:
What base?

Reply:
FreeBSD!  "base" is short for "base operating system" and is the stuff that's made by the FreeBSD Project - kernel and world.  This includes everything outside of `/usr/home` and `/usr/local`, which isn't much.

A mirror of the source repo is available here, so you can see what all is part of FreeBSD's base: https://github.com/freebsd/freebsd-src